scripts/pvscheck.sh

```diff
@@ -1,12 +1,14 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
bash isn't available on some bsd systems, so it's good to avoid it unless there is a strong reason not to.